src/services/executionEnvironment.ts

```diff
@@ -310,7 +310,7 @@ export class ExecutionEnvironment {
     if (this._container_engine === "auto") {
       for (const ce of ["podman", "docker"]) {
         try {
-          child_process.execSync(`which ${ce}`, {
+          child_process.execSync(`command -v ${ce}`, {
```

Comment:
We can modify the utils function here https://github.com/ansible/ansible-language-server/blob/main/src/utils/commandRunner.ts#L82 and use it at other places

Reply:
I will have it in a separate PR.